### ISSUE TITLE
snap event triggers undefined children error

### ISSUE THREAD

**Issue by markoshust** [leongersen/noUiSlider#544]
Receiving this error when executing a tab+drag (aka snap) event. 

```
Error: Cannot read property 'children' of undefined
TypeError: Cannot read property 'children' of undefined
    at start (nouislider.js:1213)
    at tap (nouislider.js:1302)
    at HTMLDivElement.method (nouislider.js:1133)
```

Here's the slider init:

```
      noUiSlider.create(slider, {
        behaviour: "snap",
        connect: "lower",
        direction: "rtl",
        orientation: "vertical",
        range: {
          min: 0,
          max: 100 
        },  
        start: 0
      });
```

The example works on your demo page, so is there anything that could be causing this before I dig deeper?

**Comment by markoshust** [comment]
Handles are coming up undefined when tracing

![screen shot 2015-10-30 at 9 46 50 am](https://cloud.githubusercontent.com/assets/437029/10847199/36524860-7eeb-11e5-8f25-196563bd7b71.png)

**Comment by leongersen** [comment]
What version of noUiSlider are you on? This was fixed in 8.1.0.

**Comment by markoshust** [comment]
Ah -- yep :+1: I'm using this with Meteor (hence the NPM needs) and it was loading a local/cache version of 8.0.2. Error went away with 8.1.0.

FYI I created a Meteor package for this that works from the new NPM package. https://github.com/markoshust/meteor-nouislider -- https://atmospherejs.com/markoshust/nouislider

I don't think Meteor has the ability to give other owners access, but I'll be sure to maintain this package. It's really just a skeleton. You're also free to fork & maintain your own Atmosphere/Meteor package from my github repo if you wish, then I'll remove mine from Atmosphere. There's also something new starting to alleviate this pain that allows you to autopublish Meteor packages on GitHub pushes https://autopublish.meteor.com/howitworks
